src/cli/__tests__/config.test.ts

```diff
@@ -12,10 +12,7 @@ import { loadProjectStructure } from './loadProjectStructure';
 jest.mock('../log');
 
 afterEach(() => {
-  // clear require cache
-  Object.keys(require.cache).forEach((modulePath) => {
-    delete require.cache[modulePath];
-  });
+  jest.resetModules();
```

Comment:
Nice.

Reply:
oh I should mention that I found out last night that jest's `require` doesn't actually implement the `require.cache`.